hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -35,12 +48,35 @@ Check that the output `0x3020000a` is shown. If it is not, then the OTP bit has
 
 If you wish, you can remove the `program_usb_boot_mode` line from config.txt, so that if you put the SD card into another Raspberry Pi, it won't program USB host boot mode. Make sure there is no blank line at the end of config.txt.
 
-## Prepare the USB mass storage device
+You can now boot from an USB Mass Storage device in the same way as booting from SD, see the following paragraph for Raspberry Pi 3B+, CM3+.
+
+<a name="pi3plus"></a>
+## Raspberry Pi 3B+, CM3+
+
+The Raspberry Pi 3B+ and CM3+ support USB Mass Storage boot out of the box. The settings specific to the previous versions of Raspberry Pi do not have to be executed.
+
+This is the same [procedure](../../../installation/installing-images) as for SD cards.
+
+After preparing the storage device, connect the drive to the Raspberry Pi and power up the Pi (be aware of the extra USB power usage from external drive).
+After five to ten seconds, the Raspberry Pi should begin booting and show the rainbow splash screen on an attached display. Make sure that you do not have an SD card inserted in the Pi, since if you do, it will boot from that first.
+
+<a name="unsupported"></a>
+## Unsupported devices
+
+Including Raspberry Pi 1, 2 (V1.1), Compute Module, Zero.
+
+The boot code for USB Mass Storage boot is stored in the BCM2837(B0)'s ROM only. Earlier versions such as the BCM2836 and BCM2835 do not have this version of bootcode with USB Mass Storage boot.
+
+Since up to the Pi 4B, the boot code is written in ROM (Read only memory) it cannot be updated. For this reason the Pi 1 A/B, Pi 2 B (v1.1), and Pi Zero will all require SD cards.
+
```

Comment:
"since up to the Pi 4B" - sounds odd, and please use the entire (tedious I know) Pi name e.g. "From the Raspberry Pi 4B onwards" or something.

Reply:
I don't think you need the "unsupported devices" section, since the page already specifies exactly which devices MSD boot works with at the very top.

Reply:
I agree on the "Unsupported devices", it's a bit of an odd heading for it; as well as it gives a lot of info about devices for which this page isn't really relevant. And the reasoning is also documented on the "Special bootcode.bin-only boot mode page".

I did move the reference to the 'special bootcode.bin-only boot mode' to the intro; so that any user can still find it's way to an alternative solution for his Raspberry Pi if it's not supported.

Also to still comply with  #1361 

By that the whole "Since up to" is gone as well :)

Reply:
Also; I've changed the capitalisation from "USB Mass Storage Boot mode" to "USB mass storage boot". For some reason it doesn't feel better; but it does seem more correct.

Reply:
> Also; I've changed the capitalisation from "USB Mass Storage Boot mode" to "USB mass storage boot". For some reason it doesn't feel better; but it does seem more correct.

Without the capitalisation is correct. Originally there was a page that called it MSD boot, so that's probably how the capitalisation crept in.